<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizationWindow</class>
 <widget class="QMainWindow" name="VisualizationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>862</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,25">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="visualizationArea">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="sceneRect">
         <rectf>
          <x>0.000000000000000</x>
          <y>0.000000000000000</y>
          <width>2000.000000000000000</width>
          <height>5000.000000000000000</height>
         </rectf>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::RubberBandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorViewCenter</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::SmartViewportUpdate</enum>
        </property>
        <property name="optimizationFlags">
         <set>QGraphicsView::DontAdjustForAntialiasing</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="zoomSlider">
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="pageStep">
         <number>4</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="stat4Label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="stat3Label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="stat2Label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="stat1Label">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <property name="indent">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1199</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>Network</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Load</string>
     </property>
     <addaction name="actionOpen_file"/>
     <addaction name="separator"/>
     <addaction name="actionExampleNet1"/>
    </widget>
    <addaction name="menuLoad"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionDisplay_layers"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionNewLayer">
   <property name="text">
    <string>Layer</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionZmiana">
   <property name="text">
    <string>Zmiana</string>
   </property>
  </action>
  <action name="actionTrainingDialog">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Training</string>
   </property>
  </action>
  <action name="actionStats">
   <property name="text">
    <string>Stats</string>
   </property>
  </action>
  <action name="actionFunkcja_sinus">
   <property name="text">
    <string>Funkcja sinus</string>
   </property>
  </action>
  <action name="actionSine">
   <property name="text">
    <string>Sine</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Animation</string>
   </property>
  </action>
  <action name="actionDisplay">
   <property name="text">
    <string>Display</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionDisplayTrainingData">
   <property name="text">
    <string>Display</string>
   </property>
  </action>
  <action name="actionDisplay_layers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display layers</string>
   </property>
  </action>
  <action name="actionOutput_plot">
   <property name="text">
    <string>Output plot</string>
   </property>
  </action>
  <action name="actionNetwork_error">
   <property name="text">
    <string>Network error plot</string>
   </property>
  </action>
  <action name="actionOutput_plot_2">
   <property name="text">
    <string>Output plot</string>
   </property>
  </action>
  <action name="actionLive_output_plot">
   <property name="text">
    <string>Live output plot</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionExampleNet1">
   <property name="text">
    <string>Example networks</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
